### PR TITLE
Decorator improvement

### PR DESCRIPTION
**Pull request by OlehKSS**
#### What does this implement/fix?
Improvement to docs decorator, no need to provide base classes as parameters.

#### Additional information
Implemented during 2019 MNE Sprint

### CODE REVIEW

File: tutorials/test_volume_source_space.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import numpy as np  # noqa
+from mayavi import mlab  # noqa
+
```
Comment (larsoner) [inline tutorials/test_volume_source_space.py]: Errant commit to include this?

Comment (OlehKSS) [reply tutorials/test_volume_source_space.py]: Working on finding where does this thing come from

Comment (OlehKSS) [reply tutorials/test_volume_source_space.py]: Fixed

**Comment by larsoner** [comment]
Thanks @OlehKSS

**Pull request merged**